.github/workflows/release.yml

```diff
@@ -21,6 +52,15 @@ jobs:
       - name: Check out the repo
         uses: actions/checkout@v3
 
+      - name: Creating temp folder
+        run: mkdir -p target/
+
+      - name: Getting artifact
+        uses: actions/download-artifact@v3
+        with:
+          name: service
+          path: target/
+
```

Comment:
Why we need a new workflow?

Reply:
As we are building outside docker, I prefer to do it in a separate step/job. Once is done, the artifact is then saved to be downloaded during the docker build step